<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reclist generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PhonemeTab">
       <attribute name="title">
        <string>Phonemes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Separate each phoneme with a space</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="VowelsInput">
          <property name="placeholderText">
           <string>Vowels...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UnvoicedConsonantInput">
          <property name="placeholderText">
           <string>Unvoiced Consonants...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="VoiceConsonantInput">
          <property name="placeholderText">
           <string>Voiced Consonants...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="StartConsonantInput">
          <property name="placeholderText">
           <string>Start-only consonants (must be listed above)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EndConsonantIput">
          <property name="placeholderText">
           <string>End-only consonants (must be listed above)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="StartClusterInput">
          <property name="placeholderText">
           <string>Starting clusters (eg skr)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EndClusterInput">
          <property name="placeholderText">
           <string>Ending clusters (eg tSt)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SettingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>386</width>
             <height>561</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Maximum Syllables</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="MaxSyllablesInput"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Style</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>16</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>16</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QRadioButton" name="RecordingStyleRadioButtonOne">
                 <property name="text">
                  <string>- ka ka k -</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="RecordingStyleRadioButtonTwo">
                 <property name="text">
                  <string>- ka ki ku ke ko k -</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="RecordingStyleRadioButtonThree">
                 <property name="text">
                  <string>a ta da pa ba sa za</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="CVStartFrame">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="GenerateStartCVCheckBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Also generate [- CV] list</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="CVIncludeClustersCheckBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Include Clusters</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="VCEndingFrame">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="GenerateEndVCCheckBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Also generate [VC -] list</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="VCIncludeClustersCheckBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Include Clusters</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="NumberLinesCheckBox">
              <property name="text">
               <string>Number each line</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="NumberLinesPanel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>16</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>16</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QRadioButton" name="PrefixLinesRadioButton">
                 <property name="text">
                  <string>Prefix</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="SuffixLinesRadioButton">
                 <property name="text">
                  <string>Suffix</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="PrefixOfNumber">
                 <property name="placeholderText">
                  <string>Prefix for number (eg _ )</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="SuffixUppercasePlusCheckbox">
              <property name="text">
               <string>Suffix uppercase characters with a +</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="PrefixLinesWithUnderscoreCheckbox">
              <property name="text">
               <string>Prefix each line with an underscore</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>oto settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>16</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>16</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_6">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>sample offset</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="OtoOffsetInput">
                    <property name="maximum">
                     <number>99999999</number>
                    </property>
                    <property name="value">
                     <number>64</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_7">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>recording bpm</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="OtoBpmInput">
                    <property name="maximum">
                     <number>99999999</number>
                    </property>
                    <property name="value">
                     <number>120</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="SpaceSeparatedPhonemesRadioButton">
                 <property name="text">
                  <string>Separate phonemes with spaces</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="NoSpacesRadioButton">
                 <property name="text">
                  <string>Do not separate phonemes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="OnlySpaceSeparateVCRadioButton">
                 <property name="text">
                  <string>Only separate [V C] with spaces</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="DoubleAliasRadioButton">
                 <property name="text">
                  <string>Alias phonemes both space-separated and non-space-separated</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="GeneratePureUnvoicedConsonantCheckbox">
              <property name="text">
               <string>Generate pure unvoiced CCs</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="GenerateStartingConsonantsCheckBox">
              <property name="text">
               <string>Generate [- C]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="GenerateEndingConsonantsCheckbox">
              <property name="text">
               <string>Generate [C -]</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>72</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="GenerateReclistButton">
         <property name="text">
          <string>Generate list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GenerateBaseOtoButton">
         <property name="text">
          <string>Generate base oto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="LoadConfigMenuButton"/>
    <addaction name="SaveConfigAsMenuButton"/>
    <addaction name="SaveConfigMenuButton"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="LoadConfigMenuButton">
   <property name="text">
    <string>Load config...</string>
   </property>
  </action>
  <action name="SaveConfigAsMenuButton">
   <property name="text">
    <string>Save config as...</string>
   </property>
  </action>
  <action name="SaveConfigMenuButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save current config</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
